app/repositories/CampaignRepository.scala

```diff
@@ -34,6 +34,17 @@ object CampaignRepository {
     }
   }
 
+  def deleteCampaign(campaignId: String) = {
+    try {
+      Dynamo.campaignTable.deleteItem("id", campaignId)
+    } catch {
+      case e: Error => {
```

Comment:
Better to catch `NonFatal(e)` as in general there's nothing you can do about fatal errors.